how-to/business-logic/how-to-apply-filtering-criteria.md

```diff
@@ -0,0 +1,79 @@
+---
+title: How To Apply Filtering Criteria Before Returning Data
+description: An example to apply a filtering criteria before returning data
+type: how-to
+page_title: How To Apply Filtering Criteria Before Returning Data in the {{site.bs-bl}} Layer?
+slug: bl-apply-filter
+position:
+tags: backendservices, business-logic, cloud-code
+teampulseid: 
+ticketid: 
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Business Logic</td>
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Cloud Code for Data, Cloud Functions</td>
+  </tr>
+</table>
+
+## Description
+
+**Applicable only to {{site.bl-data}}**. Frequently you need to apply a strict filter on the items returned from the data operation or determine whether a filtering expression is incoming with the request.  

```

Comment:
Consider separating the bold text as a note at the beginning of the article then removing the second warning of that kind.